lib/MakiASTConsumer.cc

```diff
@@ -1057,7 +1051,7 @@ void MakiASTConsumer::HandleTranslationUnit(clang::ASTContext &Ctx) {
                         // not expanded from the body of the same macro
                         if (!ExpandedFromBody(U)) {
                             auto Operand = U->getSubExpr();
-                            Operand = skipImplicitAndParens(Operand);
+                            Operand = Operand->IgnoreImplicit();
```

Comment:
Just curious, why do we only have to ignore implicit here and nowhere else?

Reply:
Honestly it doesn't matter if we call `ignoreImplicit()` here or not, since the following call to `inTree()` will return the same in either case since were just skipping implicit exprs and the matcher for arguments doesn't match on implicit exprs anyway. I chose to keep it pretty much arbitrarily (not sure keeping it or not has any effect on performance).